### DISCUSSION TITLE
Пропозиції

### DISCUSSION THREAD

**Discussion by Denisrt13** [J-A-A-M/ukraine_alarm_map#57] [Нові ідеі]
1. Прибрати з загального налаштування, порти ,режими  мапи, піни, бо це для користувача не потрібно
2. Якось здвинути повзунки регулювання яскравості та кольору, так як, при свайпі вниз кожен раз задівається регулювання пальцем і приходиться все заново налаштовувати
3. Додати пін для вмикання виходу, при  тривозі в своєму регіоні, щоб можна було підтягнути аудіо оповіщення тривоги
4. Можливість роботи з телеграм каналом ( прошивка, налаштування)
5. Налаштування кольору світлодіода тривоги в своєму регіоні

**Comment by v00g100skr** [comment]
1. https://github.com/v00g100skr/ukraine_alarm_map/issues/59
2. https://github.com/v00g100skr/ukraine_alarm_map/issues/60
3. https://github.com/v00g100skr/ukraine_alarm_map/issues/61
4. телеграм поки не планується
5. https://github.com/v00g100skr/ukraine_alarm_map/issues/62

**Comment by VetallRX** [comment]
Додам ще декілька пропозицій:
1. Додати підтримку датчиків температури 18B20 (хоча б два: температура в приміщенні та на вулиці).
2. Підтримку HT16K33 (VK16k33) з інтерфейсом I2C для керування семисегментними індикаторами.
Ідея в тому, що є годинник великого розміру. На його індикатор виводити час-температура в приміщенні-температура на вулиці і так по колу.

**Comment by Webcoinak47** [comment]
Додайте будь ласка підтримку esp32-2432s028r - має вбудований екран , було досить зручно.
